ste/jobStatusManager.go

```diff
@@ -61,15 +61,36 @@ func (jm *jobMgr) statusMgrClosed() bool {
 
 /* These functions should not fail */
 func (jm *jobMgr) SendJobPartCreatedMsg(msg JobPartCreatedMsg) {
-	jm.jstm.partCreated <- msg
-	if msg.IsFinalPart {
-		// Inform statusManager that this is all parts we've
-		close(jm.jstm.partCreated)
+	defer func() {
+		if recErr := recover(); recErr != nil {
+			jm.Log(common.LogError, "Cannot send message on closed channel")
+		}
+	}()
+	if jm.jstm.partCreated != nil { // Sends not allowed if channel is closed
```

Comment:
Since we're not nilling things out anymore, let's remove the nil checks, they're unnecessary now.

Reply:
`partCreated` is still being nil, should I remove that and others?

Reply:
Where is that being niled? I had to remove the nil setting below to fix the sync test timeout

Reply:
Yes just double checked it's removed